client/layout/sidebar/menu.jsx

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const SidebarMenu = ( { children } ) => (
+	<li className="sidebar__menu">{ children }</li>
```

Comment:
@ockham working a bit on this one here: https://github.com/Automattic/wp-calypso/pull/2419